ibft/instance.go

```diff
@@ -260,7 +259,7 @@ func (i *Instance) SignAndBroadcast(msg *proto.Message) error {
 		SignerIds: []uint64{i.ValidatorShare.NodeID},
 	}
 	if i.network != nil {
-		return i.network.Broadcast(signedMessage)
+		return i.network.Broadcast(i.ValidatorShare.PublicKey.Serialize(), signedMessage)
```

Comment:
@nivBlox why we need to pass public key? not better to manage it with identifier?

Reply:
to get the topic